dom.bs

```diff
@@ -209,27 +209,62 @@ against a <var>set</var>, run these steps:
 added.
 
 
-<h3 id=namespaces>Namespaces</h3>
+<h3 id=name-validation oldids=namespaces>Name validation</h3>
 
-<p>To <dfn export>validate</dfn> a <var>qualifiedName</var>, <a>throw</a> an
-"{{InvalidCharacterError!!exception}}" {{DOMException}} if <var>qualifiedName</var> does not match
-the <code><a type>QName</a></code> production.
+A [=string=] |string| is a <dfn>valid basic name</dfn> if it does not contain
+[=ASCII whitespace=], U+0000 NULL, U+002F (/), or U+003E (>).
```

Comment:
Somehow `<` still makes me nervous, but as you point out, it's already allowed, so ok.